### DISCUSSION TITLE
Welcome to ModernJava Discussions!

### DISCUSSION THREAD

**Discussion by Zabuzard** [Together-Java/ModernJava#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by JJeeff248** [comment]
Hello! I am going into my 3rd year of university studying computer science and data science. I mainly lurk in the community, but I did contribute to the role selection command and am looking to contribute to this book :)

**Reply by Zabuzard** [reply]
Awesome to hear, thanks! 😃 Will probably take us a few weeks until the project is "ready" for writing actual content.
